{"level":50,"time":1600462221091,"pid":63674,"hostname":"Scotts-MBP.lan","msg":"Error: Error: ER_BAD_FIELD_ERROR: Unknown column 'c.category_i' in 'on clause'"}
{"level":50,"time":1600462340708,"pid":63674,"hostname":"Scotts-MBP.lan","msg":"Error: Error: ER_BAD_FIELD_ERROR: Unknown column 'c.category_i' in 'on clause'"}
{"level":50,"time":1600462358910,"pid":63689,"hostname":"Scotts-MBP.lan","msg":"Error: Error: ER_BAD_FIELD_ERROR: Unknown column 'c.category_i' in 'on clause'"}
{"level":50,"time":1606940640522,"pid":59882,"hostname":"Scotts-MBP.lan","msg":"Drive API Error: Error: Invalid Credentials"}
{"level":50,"time":1606951980483,"pid":60234,"hostname":"Scotts-MBP.lan","msg":"Drive API Error: Error: Project 781599762400 is not found and cannot be used for API calls. If it is recently created, enable Drive API by visiting https://console.developers.google.com/apis/api/drive.googleapis.com/overview?project=781599762400 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry."}
